tests/test_config.py

```diff
@@ -1684,7 +1683,7 @@ def test_passing_empty_string(self, tmpdir, newconfig):
         args = ['--hashseed', '']
         self._check_testenv(newconfig, '', args=args)
 
-    @pytest.mark.xfail(sys.version_info >= (3, 2),
+    @pytest.mark.xfail("sys.version_info >= (3, 2) and sys.version < (3, 4)",
```

Comment:
`sys.version_info` - `sys.version` is a string

Reply:
ups - thanks.